scripts/Application_InnoSetup.iss

```diff
@@ -62,7 +62,7 @@ Source: "support_files\msvcr100.dll"; DestDir: "{app}\enlighten\"; Flags: recurs
 Source: "support_files\libusb_drivers\amd64\libusb0.dll"; DestDir: "{app}\enlighten\"; Flags: recursesubdirs ignoreversion
 
 ; Files and directories to be copied as-is into the destination directory tree
-Source: "..\enlighten\assets\stylesheets\default\*"; DestDir: "{app}\enlighten\enlighten\assets\stylesheets\default\"; Flags: recursesubdirs ignoreversion
+Source: "..\enlighten\assets\stylesheets\*"; DestDir: "{app}\enlighten\enlighten\assets\stylesheets\"; Flags: recursesubdirs ignoreversion
```

Comment:
Hi @samiebee43, I think this is the bug you ran into when running bootstrap.  My bad.